.circleci/config.yml

```diff
@@ -0,0 +1,45 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.12.0
+    working_directory: ~/lansingcodes/lubot
+    environment:
+      TZ: America/Detroit
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dep-{{ .Branch }}-
```

Comment:
Do you mind changing the keys to:

```
        - base-deps-v1-{{ .Branch }}-{{ checksum "package-lock.json" }}
        - base-deps-v1-{{ .Branch }}-
        - base-deps-v1-master-
        - base-deps-v1-
```

This will allow us to more directly correlate caching to dependency changes.